helpers/handlebars/helpers/index.js

```diff
@@ -6,6 +6,16 @@ const { Configuration } = require('@hpi-schul-cloud/commons');
 const permissionsHelper = require('../../permissions');
 const i18n = require('../../i18n');
 const Globals = require('../../../config/global');
+const path = require('path');
+const fs = require('fs');
+
+let themeAssetDir = path.join(__dirname, '../../..', `build/${Globals.SC_THEME}`);
+if (!fs.existsSync(themeAssetDir)) {
+	// fallback to static dir, for example for test env
+	themeAssetDir = path.join(__dirname, '../../..', 'static');
+}
+const staticify = require('staticify')(themeAssetDir);
```

Comment:
creation of static files should happen in build process is this possible? 
all running pods need to have the same hash (ok, it's md5)...

Reply:
this just uses the files already generated in the build process., so they are the same on all pods. 

There might be edgewise during rolling rollouts, but this case be investigated later